lib/modules/manager/dockerfile/extract.ts

```diff
@@ -283,6 +283,25 @@ export function extractPackageFile(
       instruction += '\n' + lineLookahead;
     }
 
+    const syntaxRegex = regEx('^#[ \\t]*syntax=(?<image>\\S+)', 'm');
```

Comment:
```suggestion
    const syntaxRegex = regEx('^#[ \\t]*syntax[ \\t]*=[ \\t]*(?<image>\\S+)', 'im');
```

- It is [valid](https://docs.docker.com/engine/reference/builder/#parser-directives) and common (see [here](https://github.com/search?q=path%3A**%2FDockerfile+%22%23+syntax+%3D%22&type=code&ref=advsearch)) to use spaces before and after `=`
- `# sYnTaX = docker/dockerfile:1` is also valid -> match case-insensitive

Reply:
thanks for docs reference